vivarium_public_health/risks/base_risk.py

```diff
@@ -1,47 +1,40 @@
-import numpy as np
 import pandas as pd
-from scipy.stats import multivariate_normal, norm
 
 from vivarium.framework.randomness import random
 
-from ceam_public_health.risks import RiskEffectSet, get_distribution
+from vivarium_public_health.risks import RiskEffectSet, get_distribution
 
 
 def uncorrelated_propensity(population, risk_factor):
     return random(f"initial_propensity_{risk_factor}", population.index)
 
-class ContinuousRiskComponent:
-    """A model for a risk factor defined by a continuous value. For example
-    high systolic blood pressure as a risk where the SBP is not dichotomized
-    into hypotension and normal but is treated as the actual SBP measurement.
 
-    Parameters
-    ----------
-    risk : str
-        The name of a risk factor
-    distribution_loader : callable
-        A function which take a builder and returns a standard CEAM
-        lookup table which returns distribution data.
-    exposure_function : callable
-        A function which takes the output of the lookup table created
-        by distribution_loader and a propensity value for each simulant
-        and returns the current exposure to this risk factor.
-    """
+class RiskComponent:
+    """A model for a risk factor defined by either a continuous or a categorical value. For example,
+    (1) high systolic blood pressure as a risk where the SBP is not dichotomized
+    into hypotension and normal but is treated as the actual SBP measurement.
+    (2) smoking as two categories: current smoker and non-smoker.
 
+       Parameters
+       ----------
+       risk_type : str
+           'risk_factor'
+       risk_name : str
+           The name of a risk
+       """
 
     def __init__(self, risk_type, risk_name):
         self._risk_type, self._risk = risk_type, risk_name
         self._effects = RiskEffectSet(self._risk, risk_type=self._risk_type)
 
     def setup(self, builder):
-
         self.propensity_function = uncorrelated_propensity
 
         self.exposure_distribution = get_distribution(self._risk, self._risk_type, builder)
         builder.components.add_components([self._effects, self.exposure_distribution])
         self.randomness = builder.randomness.get_stream(self._risk)
         self.population_view = builder.population.get_view(
-            [self._risk+'_exposure', self._risk+'_propensity', 'age', 'sex'])
+            [self._risk + '_exposure', self._risk + '_propensity', 'age', 'sex'])
```

Comment:
Good for fixing old style issues.  What do you think about making stuff like this into f-strings?  E.g. `f'{self._risk}_exposure'` instead of `self._risk + '_exposure'`.